UPGRADE.md

```diff
@@ -34,6 +34,8 @@ After bundle install, you should also install npm packages:
 npm install
 ```
 
+This release changes how cookies are handled. Previously, all cookies were shared between different subdomains. Now these are separated by subdomain. If the default `session_key` has been changed in `config.defaults.yml` this requires actions. The `session_key` config variable should be renamed to `old_session_key`, and `session_key` should contain a differing key.
```

Comment:
I read this twice but can't say fully get it. So if I have session_key set I must generate myself a new key and update the value of the session_key config variable? Why is this? What is the effect of renaming current session_key to old_session_key and why must that be done?